pkg/db/migration.go

```diff
@@ -41,7 +49,7 @@ func (d *DatabaseMigrator) Migrate(ctx context.Context) error {
 	lg := log.WithFields(logging.Fields{
 		"direction": "up",
 	})
-	err := MigrateUp(d.params)
+	err := MigrateUp(d.params, false)
```

Comment:
Migrate function is called only on setup (new installation of lakeFS), so basically you should already load in KV format. It will mean that the old migration flow (which initially creates the tables) should have been removed and we will go straight to creating the kv DB.
To emulate it, I can check for KV enabled on setup and if so, skip the normal migrate. WDYT?

Reply:
I think we are OK as it is. Thanks for the explanation